contracts/MANACrowdsale.sol

```diff
@@ -27,6 +27,8 @@ contract MANACrowdsale is WhitelistedCrowdsale, CappedCrowdsale, FinalizableCrow
 
     event WalletChange(address wallet);
 
+    event PreferentialRateChange(address buyer, uint256 rate);
```

Comment:
WDYT about making buyer indexable, in order to track all changes relevant to a buyer?